sections/homepage/Assets/Assets.tsx

```diff
@@ -208,10 +209,10 @@ const Assets = () => {
 					key: market.asset,
 					name: market.asset[0] === 's' ? market.asset.slice(1) : market.asset,
 					description: description.split(' ')[0],
-					price: market.price.toNumber(),
+					price: marketPrice.toNumber(),
```

Comment:
Need to index by asset here:
![image](https://user-images.githubusercontent.com/10401554/207665185-aaa47209-0f59-4bf8-bce6-f625d7fa0367.png)